mlflow/protos/service.proto

```diff
@@ -987,18 +988,6 @@ message ListArtifacts {
   }
 }
 
-// Metadata of a single artifact file or directory.
```

Comment:
I'm not a big fan of this change. Third-party MLflow service developers will need to start carrying around & compiling an additional `common.proto` file in order to implement the MLflow service API spec.

Reply:
That's fair, I had to make multiple small changes to accommodate this myself. 
I initially created this to avoid duplicating the FileInfo across the tracking service and registry, and as a placeholder for any future common protos. However, on further discussion, it seems this would cause more issues than solve.
Thank you for pointing this out, reverting this change.